doc/radosgw/layout.rst

```diff
@@ -178,22 +183,22 @@ Known pools:
   prodtx$prodt.buckets
   test2
 
-.users.email
+  namespace: users.email
   Unimportant
 
-.users
+  namespace: users.keys
```

Comment:
while you're here, could you please this text:
`It's unclear why user ID is not used to name objects in this pool.`
with this:
`This allows radosgw to look up users by their access keys during authentication.`